contents/blog/the-posthog-array-1-32-0.md

```diff
@@ -0,0 +1,133 @@
+---
+date: 2022-01-31
+title: Array 1.32.0
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["Release notes"]
+featuredImage: ../images/blog/posthog-array-blog.png
+excerpt: Excerpt coming soon.
+---
+
+<blockquote class='warning-note'>
+<b>IMPORTANT!</b> Do not upgrade to this version if you have deployed PostHog using Postgres. PostHog no longer supports Postgres as of <code>v1.31.0</code> (last version supported is <code>1.30.0</code>) and you must <a href="/docs/self-host/migrate-from-postgres-to-clickhouse" target="_blank">upgrade to ClickHouse</a> first.
+</blockquote>
+
+## PostHog 1.32.0 release notes
+
+> Don't see the new features on your self-hosted deployment? Remember to [update your PostHog instance](/docs/self-host/configure/upgrading-posthog).
+
+**Release highlights:**
+- [Redesigned Persons & Groups pages](#redesigned-persons--groups-pages)
+- [Short link sharing](#short-link-sharing)
+- [Improved insight experience](#improved-insight-experience)
+- [Funnels vertical breakdown](#funnels-vertical-breakdown)
+
+### Redesigned Persons & Groups pages
+
+We've redesigned how the Persons & Groups pages look to make it easier to find what you're looking for. You'll now be able to easily see which groups (if any) the user belongs to, and better manage the user's properties.
+
+<img src="https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/1_32_0-new-person-page.gif" alt="Example screenshot: New Person page" />
+
+<br />
+
+### Short link sharing
+
+When sharing saved insights, you will now get a short and sweet URL to share with your teammates. This means something like this: `https://app.posthog.com/insights/y3tskB1B`, instead of well.... a very, very long URL you would see before. Further, this now makes it possible to create very complex insights with multiple series and filters.
```

Comment:
- May be good to specify "Short _insight_ link sharing"
- "well.... a very, very long URL you would see before" sounds very fluffy, something more concrete could drive the point home more convincingly, e.g. specifying that previously ALL query filters were embedded in the URL.
- As a user I'd wonder "wasn't it already possible to create very complex insights?". Better to specify the problem too, i.e. the URL length limit which you could have ran into due to all the filters in the URL.